profiles/package.mask

```diff
@@ -33,6 +33,11 @@
 
 #--- END OF EXAMPLES ---
 
+# Bernd Waibel <waebbl-gentoo@posteo.net> (2023-02-18)
+# Obsolete, last consumer is gone.
+# Removal in 4 weeks. Bug #892209
```

Comment:
Please use a specific date instead. `lr-add-pmask` from mgorny-dev-scripts is your friend.

Reply:
With all due respect, IMO this doesn't make much sense for a proxy maintainer, where PRs can linger for an extended period. This PR is here for 3 weeks. If I had used a date, I would have needed to update the date in the first line and calculate a new date by adding 30 days as well. Using a calculated date just adds extra, unneeded work for such cases. Using your script is ok if the PR gets merged quickly.

Reply:
Then put YYYY-MM-DD and we'll fill it in ;-).

Reply:
Ok, I keep this in mind.
Is there a change in policy? In the past it always good to use 4 weeks or 30 days or something. All three cases refer to a specific date, so I can see no clear advantage to press people into using only one of these.

Reply:
There's not really a policy, it's just easier to work for those doing cleanups and is clearer for users reading the message. Can't even be sure what to even add 30 days to (last rite timestamp? which may not even be when it was commited wrt proxy-maint and then it could get cleaned too early), no ambiguity with a date.

Reply:
From my POV, the cleanup date isn't anything we should look too closely. It's an estimate date on when to expect the package to be gone. A few days, even a week plus or minus, I wouldn't care about. I can see no added value in having precise dates when the package **has to** be cleaned from the tree and forcing this date. It just forces treecleaners to actually check every single day for outdated packages, while IMO once or twice a week would be enough.
In case we would have an automated cleaning mechanism, it would be perfectly ok, but as long as we have humans looking into this, it just adds extra workload, without giving any added value.

Reply:
"Plus" is ok, "minus" is not. When you tell people "you have time until […]", you don't cut it short with a shrug.